enterprise/frontend/src/embedding-sdk/components/private/SdkPortalContainer.tsx

```diff
@@ -0,0 +1,22 @@
+import styled from "@emotion/styled";
+
+import { EMBEDDING_SDK_PORTAL_ROOT_ELEMENT_ID } from "embedding-sdk/config";
+
+import { PublicComponentStylesWrapper } from "./PublicComponentStylesWrapper";
+
+/**
+ * This is the portal container used by popovers modals etc, it is wrapped with PublicComponentStylesWrapper
+ * so that it has our styles applied.
+ * Mantine components needs to have the defaultProps set to use `EMBEDDING_SDK_PORTAL_CONTAINER_ELEMENT_ID` as target for the portal
+ */
+export const PortalContainer = () => (
+  <PublicComponentStylesWrapper>
```

Comment:
could we put these properties on the `PublicComponentStylesWrapper`? I'm wary of having extra divs that might cause layout issues

Reply:
I tried that but it didn't work as `PublicComponentStylesWrapper` has other properties like "width:100%" etc, we'd have to override them if we used only one element otherwise the container would be position fixed with 100% height and width.
Unfortunately I think keeping them separate is the "safest" way as they're less coupled